src/components/Popover/Popover.module.scss

```diff
@@ -49,6 +49,28 @@
         }
     }
 
+    // Controlled popover
+    .controlledPopover {
+        display: none;
+    }
+
+    .helper {
```

Comment:
@atmelmicro, discuss this with @adamkudrna. I think it is necessary to comment this hack at the beginning of the file and add link to it.

Reply:
I agree.

@atmelmicro Please drop comments in both CSS and JS and explain why we decided to use this hack.

Reply:
@atmelmicro, @adamkudrna uses a bit different approach for commenting SCSS files using references to comments written ahead of the code (at the top of the file). Please follow the convention unless @adamkudrna states anything else.